snakebite/config.py

```diff
@@ -92,19 +93,27 @@ def read_hdfs_config(cls, hdfs_site_path):
     @classmethod
     def get_external_config(cls):
         if os.environ.get('HADOOP_HOME'):
-            configs = cls.get_config_from_env()
-            return configs
-        else:
-            # Try to find other paths
-            configs = []
-            for core_conf_path in cls.core_try_paths:
-                configs = cls.read_core_config(core_conf_path)
-                if configs:
-                    break
-
-            for hdfs_conf_path in cls.hdfs_try_paths:
-                tmp_config = cls.read_hdfs_config(hdfs_conf_path)
-                if tmp_config:
-                    # if there is hdfs-site data available return it
-                    return tmp_config
-            return configs
+            hdfs_path = os.path.join(os.environ['HADOOP_HOME'], 'conf', 'hdfs-site.xml')
```

Comment:
I thought about it initially, but eventually decided it would be an overkill, because:
* One of the usages (in `get_config_from_env`) is becoming deprecated and not used anywhere else in the snakebite - I only left it there in case someone is using that method externally
* Other two usages are slightly different from each other, i.e. the presence of `conf` part of the path